File: 프로그래머스/unrated/181839. 주사위 게임 1/주사위 게임 1.java
class Solution {
    public int solution(int a, int b) {
        int answer 
            = a % 2 == 1 && b % 2 == 1 ? (a*a)+(b*b) 
            : a % 2 == 1 || b % 2 == 1 ? 2 * (a+b) 
                : Math.abs(a-b);
                return answer;
    }
}

File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.py
def num(i):
    return i % 2 == 1

def solution(a, b):
    answer = 0
    
    if num(a) and num(b): 
        answer = a * a + b * b
    elif num(a) or num(b):
        answer = 2 * (a + b)
    else:
        answer = a - b
        if answer < 0:
            answer *= -1;
    
    return answer